src/display/annotation_layer.js

```diff
@@ -3254,6 +3254,15 @@ class AnnotationLayer {
       parent: this,
     };
     for (const data of annotations) {
+      data.borderStyle ||= {
```

Comment:
Can we create this object just once, and use it for all inferred link annotations?